### PR TITLE
L1-L4: OpenStack RabbitMQ notifications for Security Groups

### PR DESCRIPTION
**Pull request by larkinscarvalho**

### CODE REVIEW

File: designs/l1_l4_security/l1_l4_security.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+This document describes the notifications available for OpenStack security groups
+
+## Assignees
+Intel's OSC core team
+
+## Background
+OSC will support single page view to manage OpenStack security groups. User will be able to manage the OpenStack security groups and security group rules from OSC.
+
+## Constraints and Assumptions
+No addtional security groups are configured by user or admin directly on OpenStack.
```
Comment (emanoelxavier) [inline designs/l1_l4_security/l1_l4_security.md]: Does this mean that security groups should never be managed directly on a OpenStack environment for which OSC has a virtualization connector? What if the user does create, delete or update security groups on OpenStack? Will OSC ignore? overwrite? This assumption or constraint needs more clarity.

File: designs/l1_l4_security/l1_l4_security.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+## Notifications provided by OpenStack
```
Comment (emanoelxavier) [inline designs/l1_l4_security/l1_l4_security.md]: What is the purpose of these notifications and user permissions for the implementation of this feature? Can you provide more details? The document seems to lack something that connects this info with the overall feature. Also, can you delete 'XYZ Integration Point' and place this session at the location in the document? The _XYZ Integration Point_ is not a real header, just a placeholder for a session that describes the  communication/integration of OSC with other external endpoints and I think and fits for this.

Comment (larkinscarvalho) [inline designs/l1_l4_security/l1_l4_security.md]: The scope of the US is to research about the notifications generated for create/update/delete security group and create/delete security group rule. Also, we have mentioned about what OSC is expected to do when it receives notifications on RabbitMQ. [More info](https://github.com/larkinscarvalho/community/blob/98c1ec91579a89be3244ed5f86005ea67b7102f5/designs/l1_l4_security/l1_l4_security.md#notifications-provided-by-openstack).

File: designs/l1_l4_security/l1_l4_security.md (hunk 1 above)
Comment (arvindn05) [inline designs/l1_l4_security/l1_l4_security.md]: >admin policies to restrict the user from creating the security groups
I think while the that point clarifies that we intend to restrict creating security groups(clarify it create/edit/delete), we should still add that the OSC synchronization jobs which run based on notifications or with a set interval will get the security groups back in conformance by making sure whatever is configured in OSC is what is reflected in openstack

Comment (emanoelxavier) [inline designs/l1_l4_security/l1_l4_security.md]: Got it, so the approach is that we will overwrite.

**Pull request merged**